### ISSUE TITLE
Low performance and detection on Google Coral DevBoard

### ISSUE THREAD

**Issue by rahuladream** [tensorflow/tensorflow#28855]
<em>Just started working on google coral devboard. As I seen in blogs and official doc I need to convert FLOAT 32 bit neural network to QUANTIZED_UNIT8 i.e. 8bit neural network. </em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 19.04):
- Google Dev Board:
- TensorFlow installed from (source or binary):
- Edge TPU
- Python version:
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

As I run the official demo of the google dev board getting very low accuracy while fps is quite good, But the accuracy is no furthur than 50-60 %. 
The model is official and included into dev board itself. 
Can any one have any idea to improve accuracy or it will be same because of 8 bit neural network.

**Comment by gadagashwini** [comment]
@rahuladream Just to verify, Did you follow this [link](https://coral.withgoogle.com/docs/dev-board/get-started/). ?. Thanks!

**Comment by rahuladream** [comment]
> @rahuladream Just to verify, Did you follow this [link](https://coral.withgoogle.com/docs/dev-board/get-started/). ?. Thanks!

Yes, I did 
The official one also have low accuracy

**Comment by petewarden** [comment]
I haven't worked on this area, but I've pinged the Coral team to ask for the right owner.

**Comment by manoj7410** [comment]
@rahuladream can you please send your query at coral-support@google.com so that your question is visible to larger audience and we would be able to help you appropriately?

**Comment by gadagashwini** [comment]
@rahuladream Did you get a chance to look at @manoj7410's comment. Thanks!

**Comment by gadagashwini** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
